### DISCUSSION TITLE
Dialog comes up during "pushkin start;"

### DISCUSSION THREAD

**Discussion by tmintz** [l3atbc-datadog/pushkin#100] [General] [answered]
I did all the quickstart setup (except for setting logins -- I kept useForum and useAuth false), and ran "pushkin start;".  It ran for a while and then produced:
```
Pulling vocab_worker (vocab_worker:)...
The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]
```
Is this normal?

**Comment by jkhartshorne** [comment, marked as answer]
No, it isn't. Did you make an experiment called 'vocab'?

**Reply by tmintz** [reply]
Yes.  I made vocab, then mind, then whichenglish.  Just went back and confirmed.

**Reply by jkhartshorne** [reply]
Did you by any chance run `pushkin kill` or `pushkin armageddon` at any point? I think the error is saying that the image for part of your 'vocab' experiment has been deleted from Docker. That shouldn't really happen on its own.

But that's mostly a question because I'm wondering what happened. If you re-run `pushkin prep`, it _should_ take care of the issue. Please let me know if it doesn't.

**Reply by tmintz** [reply]
Ok, I will try that.  But no, I did not `kill` or `armageddon` at any point.

**Reply by tmintz** [reply]
Based on what you said I went back and looked at my series of commands and I realize I skipped running `pushkin prep` after installing the the three experiments.  I expect that was the problem.

**Reply by jkhartshorne** [reply]
Yep, that would be it!

**Comment by kittymimo** [comment]
I've got the exact same problem. I tried re-running `pushkin prep` and then `pushkin start` but it doesn't change anything. Thanks for your help!

**Reply by jkhartshorne** [reply]
@kittymimo Sorry to hear that. Can you please paste in your output from `pushkin prep` and from `pushkin start`? Thank you.

**Reply by kittymimo** [reply]
No problem! I tried running `pushkin kill` and redoing the whole thing and it worked this time! So I'm not sure what went wrong the first time but everything is alright now. Thank you!

**Reply by kittymimo** [reply]
I ran into the same problem after installing the experiment lex. I checked my output and there seemed to be a problem with .DS_Store files, so I ran `rm -rf experiments/.DS_Store` and now everything is working!

**Reply by jkhartshorne** [reply]
@kittymimo - yes, that's something a few people have run into on Mac. @conbainbridge has been working on a patch for the CLI. I think she's still testing it. In the meantime, your method will work.
